sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -446,10 +443,12 @@ def set_queue_access_policy(self,
     @distributed_trace
     def send_message(
             self,
-            content,  # type: Any
-            **kwargs  # type: Any
-    ):
-        # type: (...) -> QueueMessage
+            content: Any,
```

Comment:
Leaving content type as `Any` for now. Docstring:
```
:param obj content:
            Message content. Allowed type is determined by the encode_function
            set on the service. Default is str. The encoded message can be up to
            64KB in size.
```

Reply:
That's fine for now. We can maybe do something fancy with `overload` at some point. Or at least come up with the Union of all the possible types it could be.